src/app/src/main/kotlin/org/imperial/mrc/hint/models/ErrorReport.kt

```diff
@@ -0,0 +1,18 @@
+package org.imperial.mrc.hint.models
+
+import java.time.Instant
+
+data class ErrorReport(
+        val email: String,
+        val country: String? = "",
```

Comment:
Normally I'd prefer to leave these as null and let the JSON serialiser elide them from the request sent to the Flow. That feels more natural than trying to represent an unset value as an empty string.

But I guess that sending empty strings makes it easier to write the Flow?